### ISSUE TITLE
Сan entt store a vector?

### ISSUE THREAD

**Issue by 0LGL0** [skypjack/entt#1007]
hello again :), I would like to know if it is possible to contain std::vector in my entity, just when I declare a vector in the entity class (I think you remember this class from my last problem), an error appears. And although the error generally says about another library (ImGui), but it seems to me that entt cannot store a vector

**Comment by skypjack** [comment]
It's hard to help without a repro. Even more if you mention a generic error without specifying what you're talking about.
To be honest, I don't remember your implementation of the entity class. Nor do I know what it means to _declare a vector in it_.
However, I'm pretty sure that EnTT supports standard types on components as well as anywhere else.

As it stands, this issue is invalid. There is no way I can help here. I'm sorry.
Feel free to add more details. The best would be a repro though. I can't assure you that I can help in all other cases.

**Comment by 0LGL0** [comment]
Ok, here is the full implementation of the "Entity" class: https://gist.github.com/0LGL0/27a588c0e877c8ed0edae196b4639614 . And when I add std::vector<uint32_t> m_group to this class (this vector should store the ids of child entities for this entity), an error appears, although, again, something like this is output in the console: there is no TreePop/PopStyle from ImGui or something like this (if it is not clear, then I apologize, now I'm not near the computer)

**Comment by skypjack** [comment]
Wait, is this the same entity that you use with the registry? As in `using my_registry = basic_registry<Entity>;`?

**Comment by skypjack** [comment]
Let me rephrase it then 🙂 what's `m_reg`? Can you tell me the exact type of this member?

**Comment by 0LGL0** [comment]
m_reg is a type of entt::registry, sorry I didn't understand right away

**Comment by 0LGL0** [comment]
Ok, I solved this problem by simply creating a new component "GroupComponent", now everything seems to work.
